### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by damusnet**

### CODE REVIEW

**Review by timothychin — APPROVED** [review]

File: adobe_premiere_plugin/README.md [hunk 1]
```diff
@@ -1,19 +1,91 @@
 # Adobe Premiere Plugin
 
-_Coming Soon_ – An Adobe Premier Plugin that brings the search power of
-Scrounger into Adobe Premiere is in development. The plugin will allow editors
-to access Scrounger natively in Premiere and import the results of their
-searches directly into their projects. Check the issues in the Github repo for
-updates on progress.
+An plugin that brings the search power of Scrounger into Adobe Premiere Pro.
```
Comment (octaviaaris) [inline adobe_premiere_plugin/README.md:3]:
```suggestion
A plugin that brings the search power of Scrounger into Adobe Premiere Pro.
```

File: adobe_premiere_plugin/README.md [hunk 2]
```diff
@@ -1,19 +1,91 @@
...
+This allows editors to access Scrounger natively in Adobe Premiere Pro and
+import the results of their searches directly into their projects.
+
+## How It Works
+
+The panel is a simple static html page that lets the user fill in the location
```
Comment (octaviaaris) [inline adobe_premiere_plugin/README.md:9]:
```suggestion
The panel is a simple static html page that lets the user fill in the url
```

File: adobe_premiere_plugin/README.md [hunk 3]
```diff
@@ -1,19 +1,91 @@
...
+of their Scrounger application on the web, and the location to save the imported
+assets on their computer. Once the user clicks “Go”, they are redirected to the
```
Comment (octaviaaris) [inline adobe_premiere_plugin/README.md:11]:
```suggestion
assets on their computer. Once the user clicks “Go,” they are redirected to the
```

**Review by octaviaaris — APPROVED** [review]

File: adobe_premiere_plugin/README.md (hunk 3 above)
Comment (damusnet) [reply adobe_premiere_plugin/README.md:11]: I don't understand this one 🤔

Comment (octaviaaris) [reply adobe_premiere_plugin/README.md:11]: it's just a grammar thing. commas go inside closing quotes.

Comment (damusnet) [reply adobe_premiere_plugin/README.md:11]: Thanks, I hate it ☹️

**Pull request merged**